map_coverage_offset: 0.1
map_coverage_margin: 0.9
landmine_radius_area: 1.0
waypoints:
  manual: true
  size: 11
  wp0:
    x: 1.08414
    y: -0.203271
  wp1:
    x: 0.592041
    y: -0.198308
  wp2:
    x: -0.127722
    y: -0.181265
  wp3:
    x: 2.16829
    y: -0.406542
  wp4:
    x: -3.26997
    y: -0.405549
  wp5:
    x: 4.33658
    y: -0.813085
  wp6:
    x: -9.8099
    y: -1.21665
  wp7:
    x: -10.1883
    y: 3.16767
  wp8:
    x: 10.2281
    y: 4.03173
  wp9:
    x: 9.7573
    y: -1.82944
  wp10:
    x: 0.0
    y: 0.0
